loot_importer.user.js

```diff
@@ -126,50 +156,70 @@ function ParseLootPage() {
 
   // div#productContent table tbody tr.productOverview td.productInfo h1
   var AlbumName = document.querySelectorAll("div#productContent > table > tbody > tr.productOverview > td.productInfo > *");
-  LOGGER.debug("Album Name: ", AlbumName[0].innerText); // textContent give the space formatting and we just need the text so we use innerText
-  releasetitle = AlbumName[0].innerText;
-  LOGGER.debug("Artist Name: ", AlbumName[1].innerText);
+
   releaseartist = AlbumName[1].innerText;
+  if (releaseartist == "Various Artists") {
+    // Everything is: title(format)
+    releaseartisttitle_regex = /(.*)\((.*)\)/;
```

Comment:
I would use to ensure we match external parenthesis:
```
releaseartisttitle_regex = /(.*?)\((.*)\)/;
```

Compare:
```javascript
var s = 'A(B(C))';
var r = /(.*)\((.*)\)/;
var m = str.match(r);
console.log(m);
```
`["A(B(C))", "A(B", "C)"]`

To:

```javascript
var s = 'A(B(C))';
var r = /(.*?)\((.*)\)/;
var m = str.match(r);
console.log(m);
```

`["A(B(C))", "A", "B(C)"]`